examples/hypervisor_cluster_profiles/hypervisor_cluster_profile.tf

```diff
@@ -0,0 +1,124 @@
+provider "oneview" {
```

Comment:
just remove this file since this file is divided to other files(data_source, main, update_resource.tf), its not required.